src/functionalTest/java/steps/ClaimStatusSteps.java

```diff
@@ -0,0 +1,85 @@
+package steps;
+
+import net.thucydides.core.annotations.Step;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
```

Comment:
Organise these imports.  If you're using Visual Studio Code the shortcut is option + shift + o (assuming you are using a Mac).